_src/recipes/010-add-a-sitemap.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Add a Sitemap to 11ty site
+authorName: Akbarsait
+authorUrl: https://www.akbarsait.com
+---
+
+Sitemap helps search engine crawl for making easier and quicker way to find and index your website content. The base blog of 11ty is already equipped but here we have added few changes especially for the dates to make it easier and readable. 
+
+### Directions
+1. Create a file in your source directory  called **sitempa.njk** and add the following content to it.
```

Comment:
Also you have a typo here in the name of the file.

Reply:
Thanks. I have updated it